src/Core/src/Handlers/MenuFlyoutSubItem/MenuFlyoutSubItemHandler.Windows.cs

```diff
@@ -43,6 +43,11 @@ public static void MapText(IMenuFlyoutSubItemHandler handler, IMenuFlyoutSubItem
 			handler.PlatformView.Text = view.Text;
 		}
 
+		/// <summary>
+		///	Maps accelerator to the Windows platform control.
+		/// </summary>
+		/// <param name="handler">The handler, of type IMenuFlyoutSubItemHandler.</param>
```

Comment:
nit: you can add the cref tag to create automatic links when the documentation is autmatically made :) 
```suggestion
/// <param name="handler">The handler, of type <see cref="IMenuFlyoutSubItemHandler">.</param>
```